libbeat/kibana/client.go

```diff
@@ -214,7 +214,7 @@ func (conn *Connection) SendWithContext(ctx context.Context, method, extraPath s
 	addHeaders(req.Header, headers)
 
 	contentType := req.Header.Get("Content-Type")
-	if contentType != "multipart/form-data" && contentType != "application/ndjson" {
+	if !strings.HasPrefix(contentType, "multipart/form-data") && contentType != "application/ndjson" {
```

Comment:
[Media type parser](https://pkg.go.dev/mime#ParseMediaType) can be used to avoid surprises.
```suggestion
	contentType, _, _ = mime.ParseMediaType(contentType)
	if contentType != "multipart/form-data" && contentType != "application/ndjson" {
```